### PR TITLE
[Do not merge] Remove auto generated glog args

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Create a separate FlagSet and use it instead of automatically generate CommandLine FlagSet.

Did test with  a sample and saw that the args were passed correctly to daprd. 

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #1784 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

Output now
```
 ./dist/darwin_amd64/release/daprd --help
Usage of dapr-runtime:
  -allowed-origins string
    	Allowed HTTP origins (default "*")
  -app-id string
    	A unique ID for Dapr. Used for Service Discovery and state
  -app-port string
    	The port the application is listening on
  -components-path string
    	Path for components directory. If empty, components will not be loaded. Self-hosted mode only
  -config string
    	Path to config file, or name of a configuration object
  -control-plane-address string
    	Address for a Dapr control plane
  -dapr-grpc-port string
    	gRPC port for the Dapr API to listen on (default "50001")
  -dapr-http-port string
    	HTTP port for Dapr API to listen on (default "3500")
  -dapr-internal-grpc-port string
    	gRPC port for the Dapr Internal API to listen on
  -enable-metrics
    	Enable prometheus metric (default true)
  -enable-mtls
    	Enables automatic mTLS for daprd to daprd communication channels
  -enable-profiling
    	Enable profiling
  -log-as-json
    	print log as JSON (default false)
  -log-level string
    	Options are debug, info, warning, error, or fatal (default "info")
  -max-concurrency int
    	Controls the concurrency level when forwarding requests to user code (default -1)
  -metrics-port string
    	The port for the metrics server (default "9090")
  -mode string
    	Runtime mode for Dapr (default "standalone")
  -placement-address string
    	Address for the Dapr placement service
  -profile-port string
    	The port for the profile server (default "7777")
  -protocol string
    	Protocol for the application: grpc or http (default "http")
  -sentry-address string
    	Address for the Sentry CA service
  -version
    	Prints the runtime version
```

### CODE REVIEW

File: pkg/runtime/cli.go [hunk 1]
```diff
@@ -24,32 +24,34 @@ import (
...
-	controlPlaneAddress := flag.String("control-plane-address", "", "Address for a Dapr control plane")
-	sentryAddress := flag.String("sentry-address", "", "Address for the Sentry CA service")
-	placementServiceAddress := flag.String("placement-address", "", "Address for the Dapr placement service")
-	allowedOrigins := flag.String("allowed-origins", DefaultAllowedOrigins, "Allowed HTTP origins")
-	enableProfiling := flag.Bool("enable-profiling", false, "Enable profiling")
-	runtimeVersion := flag.Bool("version", false, "Prints the runtime version")
-	maxConcurrency := flag.Int("max-concurrency", -1, "Controls the concurrency level when forwarding requests to user code")
-	enableMTLS := flag.Bool("enable-mtls", false, "Enables automatic mTLS for daprd to daprd communication channels")
+	// Create a new FlagSet and add the flags to that.
+	f1 := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```
Comment (youngbupark) [inline pkg/runtime/cli.go:28]: can we find the better naming for `f1` ? :)

Comment (mukundansundar) [reply pkg/runtime/cli.go:28]: runtimeFlagSet ?

**Comment by mukundansundar** [comment]
Will try to fix the component that is using glog.

**Pull request closed without merging**